letsencrypt/storage.py

```diff
@@ -95,6 +100,7 @@ def __init__(self, config_filename, cli_config):
         """
         self.cli_config = cli_config
         if not config_filename.endswith(".conf"):
+            logger.debug("Renewal config file name must end in .conf")
```

Comment:
I'm not really sure what I should be including inside ```debug()```. Taking this line as an example, is what I'm doing here correct? Or should I add ```errors.CertStorageError``` as an argument instead?